conf/messages

```diff
@@ -841,27 +841,24 @@ disposalMethod.personalRep.error.invalid=Select if the person sold it, gave it a
 
 disposalMethod.cyaChange=how the property was sold or disposed of
 
-wereYouAUKResident.main.title=Tell us about your residence status on the disposal date
```

Comment:
we might as well delete these messages from the welsh file as well if they're not gonna be used anymore

Reply:
will do